packages/action-menu/src/ActionMenu.ts

```diff
@@ -55,6 +54,7 @@ export class ActionMenu extends ObserveSlotText(PickerBase, 'label') {
                     <sp-icon-more class="icon"></sp-icon-more>
                 </slot>
                 <slot name="label" ?hidden=${!this.hasLabel}></slot>
+                <slot name="tooltip"></slot>
```

Comment:
New slots need to be documented with the others around line 29. It would be great to add an example of this usage to the READEME.md, as well.